cmd/application.go

```diff
@@ -58,9 +58,14 @@ var applicationGetCmd = &cobra.Command{
 		}
 		if applicationShortFlag {
 			fmt.Print(app)
-		} else {
-			fmt.Printf("The current application is: %v\n", app)
+			return
 		}
+		if app == "" {
+			fmt.Printf("There's no active application. You need to create one\n")
```

Comment:
You can create one by running `ocdev application create <name>`

Reply:
updated